client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -79,7 +79,7 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 				<Card className="steps__form">
 					<img
 						className="steps__illustration"
-						src={ '/calypso/images/illustrations/site-title.svg' }
+						src={ '/calypso/images/illustrations/illustration-start-jetpack.svg' }
```

Comment:
I'd name this one `jetpack-start.svg` (we introduced the `jetpack-start` illustration specifically for this step, so we can just replace the file)

Reply:
👍